README.rst

```diff
@@ -76,10 +76,11 @@ Then you need to build sources with::
 
 Create a package for your distribution::
 
-      checkinstall --pkgname=caffeine-ng --pkgversion=3.4 --requires="python-docopt \(\>=0.6.2\),python-ewmh \(\>=0.1.4\),python-setproctitle \(\>=1.1.10\),python-wheel \(\>=0.29.0\),python-xdg \(\>=0.25\)" --conflicts="caffeine"
+      checkinstall --pkgname=caffeine-ng --pkgversion=3.4 --requires="python-docopt \(\>=0.6.2\),python-ewmh \(\>=0.1.4\),python-setproctitle \(\>=1.1.10\),python-wheel \(\>=0.29.0\),python-xdg \(\>=0.25\)" --conflicts="caffeine" --nodoc python ./setup.py install --install-layout=deb
 
 Replace version string with correct version and append this command with ``--install=no`` 
 should you wish to inspect created package before installing it
+Replace ``python`` with ``python3`` in package names and ``checkinstall`` invocation above if your system python is version 3
```

Comment:
I think you meant `if your system python is NOT version 3`.

Reply:
It is not a typo, but is indeed a vague statement
I have reworded it